azure-commons-core/src/main/java/com/microsoft/jenkins/azurecommons/core/credentials/TokenCredentialData.java

```diff
@@ -25,6 +26,8 @@
     private String subscriptionId;
     private String clientId;
     private String clientSecret;
+    private byte[] certificateBytes;
```

Comment:
does serializaion work on this field?

Reply:
It's working as I checked. `byte[]` is also considered as primitive type.